peekingduck/nodes/draw/bbox.py

```diff
@@ -36,24 +36,52 @@ class Node(AbstractNode):
 
         |bbox_labels_data|
 
+        |bbox_scores_data|
+
     Outputs:
         |none_output_data|
 
     Configs:
         show_labels (:obj:`bool`): **default = False**. |br|
             If ``True``, shows class label, e.g., "person", above the bounding
             box.
+        show_scores (:obj:`bool`): **default = False**. |br|
+            If ``True``, shows prediction scores, e.g., "0.99", at the bottom
+            left of the bounding box. (If set as ``True``, the previous node must
+            output |bbox_scores_data|, otherwise the pipeline willbreak.)
+        color_choice (:obj:`List[int]`): **default = None**. |br|
```

Comment:
The default is `null`, not `None`.
c.f. `dabble.statistics` node's docstrings.

Reply:
Resolved.